README.md

```diff
@@ -90,6 +90,19 @@ in the same group, messages will be balanced among them.
 // all the brokers in the cluster. ErrStreamExists is returned if a stream with
 // the given name already exists.
 client.CreateStream(context.Background(), "foo.*", "my-stream", lift.MaxReplication())
+
+// Create a stream attached to the NATS subject "foo.*" that is replicated to
+// all the brokers in the cluster. ErrStreamExists is returned if a stream with
+// the given name already exists.
+// This also set the retention.max.age for the created stream
+client.CreateStream(context.Background(), subject, name, lift.RetentionMaxAge(time.Nanosecond))
```

Comment:
Probably should set `RetentionMaxAge` to a more realistic value, such as 1 hour. I've found people often copy/paste examples and using a nanosecond for retention wouldn't be a great idea. :)